spec/facility_spec.rb

```diff
@@ -2,25 +2,254 @@
+      @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+
+      expect(@facility_1.collected_fees).to eq (100)
+      expect(@cruz.plate_type).to be (:regular)  
+    end
+
+    it 'can register an electric vehicle, collect fees, and assign a license plate' do
+      @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@bolt)
+
+      expect(@facility_1.collected_fees).to eq (200)  
+      expect(@bolt.plate_type).to be (:ev)
+    end
+  end
+
+  describe 'collected fees' do
+    it 'can collect fees on multiple vehicle registrations' do
+      @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+      
+      @facility_1.register_vehicle(@cruz)
+      @facility_1.register_vehicle(@bolt)
+      @facility_1.register_vehicle(@camaro)
+
+      expect(@facility_1.collected_fees).to eq (0)
+      
+      @facility_1.add_service('Vehicle Registration')
+      
+      @facility_1.register_vehicle(@cruz)
+      @facility_1.register_vehicle(@bolt)
+      @facility_1.register_vehicle(@camaro)
+
+      expect(@facility_1.collected_fees).to eq (325)
+    end
+  end
+
+  describe 'administer written test' do
+    it 'can administer a written test' do
+      @registrant_1 = Registrant.new('Bruce', 18, true)
+
+      @facility_1.add_service('Written test')
+      @facility_1.administer_written_test(@registrant_1)
+
+      expect(@registrant_1.license_data[:written]).to be true
+      expect(@registrant_1.license_data[:license]).to be false
+      expect(@registrant_1.license_data[:renewed]).to be false
+    end
+
+    it 'can deny a written test' do
+      @registrant_2 = Registrant.new('Penny', 15)
+
+      @facility_1.add_service('Written test')
+      @facility_1.administer_written_test(@registrant_2)
+
+      expect(@registrant_2.license_data[:written]).to be false
+      expect(@registrant_2.license_data[:license]).to be false
+      expect(@registrant_2.license_data[:renewed]).to be false
+    end
+
+    it 'can deny a written test even if registrant is of age' do
+      @registrant_3 = Registrant.new('Tucker', 16)
+
+      @facility_1.add_service('Written test')
+      @facility_1.administer_written_test(@registrant_3)
+
+      expect(@registrant_3.license_data[:written]).to be false
+      expect(@registrant_3.license_data[:license]).to be false
+      expect(@registrant_3.license_data[:renewed]).to be false
+    end
```

Comment:
For these it blocks, we want to be even more descriptive of what we're testing in each it block. This method has three conditions that must be met in order for the method to get into the if statement, so we want each it block to reflect that logic so something like `it "checks for service"` and `it "checks registrant's age"`, etc. Then we want to test all 3 conditions including if none of the conditions are met, and test the return values and side effects. You're on the right track here!